### PR TITLE
Add ReLU6 as an alias to F.clipped_relu

### PR DESCRIPTION
**Pull request by aksub99**
Add relu6 function to `clipped_relu.py`
Ref. #6376 #1448

### CODE REVIEW

**Comment by Crissman** [comment]
For WIP, you can use the new "Draft" label when submitting the PR.

**Comment by aksub99** [comment]
OK @Crissman , I'll keep that mind for future PRs.

**Comment by aksub99** [comment]
@Crissman Could you please review? If there are any changes I can make, do let me know.

File: chainer/functions/activation/clipped_relu.py [hunk 1]
```diff
@@ -155,3 +155,36 @@ def clipped_relu(x, z=20.0):
...
+    .. math:: \\text{ReLU6}(x) = \\min(\\max(0, x), 6).
+
+    Args:
+        x (:class:`~chainer.Variable` or :ref:`ndarray`):
+            Input variable. A :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
+    Returns:
+        ~chainer.Variable: Output variable. A
+        :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
```
Comment (toslunar) [inline chainer/functions/activation/clipped_relu.py]: A hyperlink could help users who want other thresholds.
```suggestion

    .. seealso:: :func:`chainer.functions.clipped_relu`

```

**Comment by aksub99** [comment]
@toslunar  I've made the suggested change.

**Comment by aksub99** [comment]
@toslunar @Crissman Are there any other modifications I could make?

File: chainer/functions/activation/clipped_relu.py [hunk 2]
```diff
@@ -155,3 +155,38 @@ def clipped_relu(x, z=20.0):
...
+    .. seealso:: :func:`chainer.functions.clipped_relu`
+
+    .. admonition:: Example
+
+        >>> x = np.random.uniform(-100, 100, (10, 20)).astype(np.float32)
```
Comment (toslunar) [inline chainer/functions/activation/clipped_relu.py]: I prefer a more concrete example, e.g.
```
>>> x = np.array([-20, -2, 0, 2, 4, 10, 100], np.float32)
>>> x
(...)
>>> F.relu(x)
Variable(...)
```

@Crissman What do you think?

Comment (aksub99) [reply chainer/functions/activation/clipped_relu.py]: @toslunar Thanks for the review. I have improved the example as you suggested. Is this fine?

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
The test failures are unrelated to the PR.
